build

```diff
@@ -1,10 +1,29 @@
 #!/bin/bash
+#
+# Build a new Python Docker image.
+#
+# Usage:
+#   ./build [--gpu] [--use-cache]
+#
+# Options:
+#   --gpu: Build an image with GPU support.
+#   --use-cache: Use layer cache when building a new image.
+# 
 set -e
+set -x
+
+CACHE_FLAG="--no-cache"
```

Comment:
Although these are modified during init from args, consider marking them as `readonly` following that:

```
readonly CACHE_FLAG
```

after your if block below.